pkg/executor/bacalhau/client.go

```diff
@@ -0,0 +1,70 @@
+package bacalhau
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/bacalhau-project/bacalhau/pkg/publicapi/apimodels"
+	retryablehttp "github.com/hashicorp/go-retryablehttp"
+)
+
+const API_VERSION = "v1"
+
+type BacalhauClient struct {
+	http    *retryablehttp.Client
+	options BacalhauExecutorOptions
+}
+
+func NewClient(options BacalhauExecutorOptions) (*BacalhauClient, error) {
+	http := retryablehttp.NewClient()
+	return &BacalhauClient{
+		http:    http,
+		options: options,
+	}, nil
+}
+
+func (client *BacalhauClient) Alive() (bool, error) {
+	result, err := getRequest[apimodels.IsAliveResponse](client, "agent/alive")
+	if err != nil {
+		return false, err
+	}
+
+	fmt.Println("%v\n", result)
```

Comment:
Could we remove this? Assuming it was for debugging during development.

Reply:
🙈